sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -330,8 +330,11 @@ public ContactDto saveContact(ContactDto dto, boolean handleChanges, boolean han
 			if (dropped || convertedToCase) {
 				contactService.cancelFollowUp(
 					entity,
-					I18nProperties
-						.getString(convertedToCase ? Strings.messageSystemFollowUpCanceled : Strings.messageSystemFollowUpCanceledByDropping));
+					new StringBuilder(entity.getFollowUpComment()).append(" ")
```

Comment:
Tests are failing. I suspect this is the reason and that StringBuilder doesn't like to be initiated with null.